### ISSUE TITLE
Text_Debug example

### ISSUE THREAD

**Issue by sigten** [bevyengine/bevy#1170]
**Bevy version**

0.4.0

**Operating system & version**

Windows 10

**What you did**

Cloned the Bevy repo, navigated to the Bevy directory, and ran:
cargo run --example text_debug --release

**What you expected to happen**

![text_layout](https://user-images.githubusercontent.com/62679105/103385738-4c0d0f80-4ac1-11eb-9b33-4ccee78bdc2e.png)
(from the Bevy 0.4.0 release notes)

**What actually happened**

![Screenshot](https://user-images.githubusercontent.com/62679105/103385742-4f080000-4ac1-11eb-97e4-2842746322a4.png)
(screenshot)

**Additional information**

- Perhaps there is an issue with the size and/or max_size fields in TextBundle?
-  I seem to be seeing the infamous wavy text bug (wasn't this resolved in 0.4.0?)
- I am also getting very poor rendering quality with text (this could from the known issue with regard to the HiDPI changes made in 0.4.0)
-  My frame rate seems to be limited to 30 fps, despite vsync being disabled in the example. I tried adding 
```
.add_resource(ScheduleRunnerSettings::run_loop(Duration::from_secs_f64(
            1.0 / 60.0,
        )))
``` 
to the app builder in the example, but it didn't change anything

Thank you all for your fine work and great discussions on this project! I apologize in advance if this is a coding error on my part or if more information is necessary. I am a hobbyist coder and this is my very first Github post.

**Comment by sigten** [comment]
update: while I was writing this issue, a commit came out that fixed most of the issues I was having (#1132).

![New_Screenshot](https://user-images.githubusercontent.com/62679105/103387567-1ddffd80-4aca-11eb-85e6-9f87e8607542.png)
I appear to be only having the odd frame limiting issue now. The text looks great!

**Comment by sigten** [comment]
Okay, I am now realizing that this issue is just me, sorry for the mistake! Enabling vsync does what it should and stops the frame limiting.
